plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftConfig.java

```diff
@@ -40,4 +43,30 @@ public RedshiftConfig setFetchSize(Integer fetchSize)
         this.fetchSize = fetchSize;
         return this;
     }
+
+    public Optional<String> getUnloadLocation()
+    {
+        return Optional.ofNullable(unloadLocation);
+    }
+
+    @Config("redshift.unload-location")
```

Comment:
What happens when this bucket is in a different region from the redshift cluster ?
Ideally we should fail the query with a relevant error message

Reply:
from https://docs.aws.amazon.com/redshift/latest/dg/r_UNLOAD.html
> REGION [AS] 'aws-region'
Specifies the AWS Region where the target Amazon S3 bucket is located. REGION is required for UNLOAD to an Amazon S3 bucket that isn't in the same AWS Region as the Amazon Redshift database.

>The value for aws_region must match an AWS Region listed in the [Amazon Redshift regions and endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html#redshift_region) table in the AWS General Reference.

>By default, UNLOAD assumes that the target Amazon S3 bucket is located in the same AWS Region as the Amazon Redshift database.

Does it make sense to take region as an optional input to allow using functionality incase s3 and redshift are in different regions.

Reply:
I've removed cross region support and added failure handling in case of cross region bucket.